packages/frontend/editor-ui/src/components/Folders/FolderBreadcrumbs.vue

```diff
@@ -43,15 +43,28 @@ const hiddenBreadcrumbsItemsAsync = ref<Promise<PathItem[]>>(new Promise(() => {
 // This will be used to filter out items that are already visible in the breadcrumbs
 const visibleIds = ref<Set<string>>(new Set());
 
-const currentProject = computed(
-	() => projectsStore.currentProject ?? projectsStore.personalProject,
-);
+// For shared context we do not show any folder breadcrumbs, only project badge
+const isSharedContext = computed(() => projectsStore.projectNavActiveId === 'shared');
+
+const currentProject = computed(() => {
+	// For shared context, return undefined to show "Shared with you"
+	if (isSharedContext.value) {
+		return undefined;
+	}
+	// Otherwise, fallback to personal project for new workflows
+	return projectsStore.currentProject ?? projectsStore.personalProject ?? undefined;
+});
 
 const projectName = computed(() => {
-	if (currentProject.value?.type === ProjectTypes.Personal) {
+	if (!currentProject.value) {
```

Comment:
Isn't `isSharedContext` enough to detect shared project? I feel like having additional value for `currentProject` is adding extra layer that can be avoided, unless I am missing something